### PR TITLE
Validate if the elements exists in the dom

### PR DESCRIPTION
**Pull request by maynor96**
Validates that the elements exist in the dom, this avoids warnings when the user is not in the module where the component is expected to be rendered.
image

### CODE REVIEW

**Comment by aimeos** [comment]
You can shorten this to e.g.:
```
el: document.querySelector('#item-address-group'),
```
Then, no error is reported because `el` is `null` and VueJS ignores it.

Also, you need to remove the files from the `.idea` folder in your PR.

**Comment by maynor96** [comment]
> You can shorten this to e.g.:
> 
> ```
> el: document.querySelector('#item-address-group'),
> ```
> 
> Then, no error is reported because `el` is `null` and VueJS ignores it.
> 
> Also, you need to remove the files from the `.idea` folder in your PR.

done!!!

**Comment by aimeos** [comment]
Very nice, good job! :-)
